colorbleed/plugins/maya/publish/validate_yeti_rig_inputs.py

```diff
@@ -24,15 +28,26 @@ def get_invalid(cls, instance):
 
         invalid = []
 
-        input_set = [i for i in instance if i == "input_SET"][0]
-        input_nodes = cmds.sets(input_set, query=True)
-        for node in input_nodes:
-            incoming = cmds.listConnections(node,
-                                            source=True,
-                                            destination=False,
-                                            connections=True,
-                                            plugs=True)
+        input_set = [i for i in instance if i == "input_SET"]
```

Comment:
Here `input_set` ends up being a list. Maybe this makes more sense as:
`input_set = next(i for i in instance if i == "input_SET")`

If you want to additionally assert instead of error.

```python
input_set = next(i for i in instance if i == "input_SET", None)
assert input_set, "Derpy"
```